> This looks fine, but again CI failed?

I'm reproducing locally 

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:testCompile (default-testCompile) on project jetty-server: Compilation failure: Compilation failure: 
[ERROR] /Volumes/workspace/dev/sources/open-sources/jetty/jetty.project/jetty-server/src/test/java/org/eclipse/jetty/server/handler/StatisticsHandlerTest.java:[37,34] cannot find symbol
[ERROR]   symbol:   class StacklessLogging
[ERROR]   location: package org.eclipse.jetty.util.log
[ERROR] /Volumes/workspace/dev/sources/open-sources/jetty/jetty.project/jetty-server/src/test/java/org/eclipse/jetty/server/handler/StatisticsHandlerTest.java:[450,14] cannot find symbol
[ERROR]   symbol:   class StacklessLogging
[ERROR]   location: class org.eclipse.jetty.server.handler.StatisticsHandlerTest
[ERROR] /Volumes/workspace/dev/sources/open-sources/jetty/jetty.project/jetty-server/src/test/java/org/eclipse/jetty/server/handler/StatisticsHandlerTest.java:[450,45] cannot find symbol
[ERROR]   symbol:   class StacklessLogging
[ERROR]   location: class org.eclipse.jetty.server.handler.StatisticsHandlerTest
[ERROR] -> [Help 1]
```

--- a/jetty-server/src/test/java/org/eclipse/jetty/server/handler/StatisticsHandlerTest.java
+++ b/jetty-server/src/test/java/org/eclipse/jetty/server/handler/StatisticsHandlerTest.java
@@ -30,11 +30,11 @@ import javax.servlet.http.HttpServletResponse;
 
 import org.eclipse.jetty.http.HttpStatus;
 import org.eclipse.jetty.io.ConnectionStatistics;
+import org.eclipse.jetty.logging.StacklessLogging;
 import org.eclipse.jetty.server.HttpChannel;
 import org.eclipse.jetty.server.LocalConnector;
 import org.eclipse.jetty.server.Request;
 import org.eclipse.jetty.server.Server;
-import org.eclipse.jetty.util.log.StacklessLogging;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;

